WordPress/Classes/Utility/Migrations/20-21/SafeReaderTopicToReaderTopic.m

```diff
@@ -8,36 +6,11 @@ @implementation SafeReaderTopicToReaderTopic
 
 - (BOOL)beginEntityMapping:(NSEntityMapping *)mapping manager:(NSMigrationManager *)manager error:(NSError **)error
```

Comment:
Yeah.   It was a one off and a big no no (for several reasons!). The situation was we needed a way to preserve the current topic when migrating. This was when the topic was being saved as an NSManagedObjectID (horrible, I know).  We're way past that point now and if someone *did* have an app so old it needed to be migrated we'd just let them have the default topic when syncing.  Yay for cleaning up the mess I guess.